src/main/java/io/r2dbc/mssql/message/tds/RoutingData.java

```diff
@@ -0,0 +1,34 @@
+package io.r2dbc.mssql.message.tds;
+
+/**
+ * Represents an alternative route.
+ */
+public final class RoutingData {
```

Comment:
How about naming this type `Redirect`, `Server` or `Endpoint`? `RoutingData` sounds a bit clunky.

Reply:
I picked {{RoutingData}} to use the same term as in the spec, but I agree. Renamed to {{Redirect}}.